src/main/java/ua/goit/timonov/enterprise/module_6_2/view/menus/StorageConsoleMenu.java

```diff
@@ -0,0 +1,150 @@
+package ua.goit.timonov.enterprise.module_6_2.view.menus;
+
+import ua.goit.timonov.enterprise.module_6_2.controllers.StorageController;
+import ua.goit.timonov.enterprise.module_6_2.model.Ingredient;
+import ua.goit.timonov.enterprise.module_6_2.model.Menu;
+import ua.goit.timonov.enterprise.module_6_2.view.console.ConsoleIO;
+
+import java.util.List;
+
+import static ua.goit.timonov.enterprise.module_6_2.view.console.ConsolePrinter.printLine;
+
+/**
+ * Console menu for tasks with ingredient storage
+ */
+public class StorageConsoleMenu extends ConsoleMenu {
```

Comment:
Totally bad design. A lot of copy-past code. I should recommend move this menu configs to some xml or something config.